docs/book.json

```diff
@@ -7,7 +7,8 @@
         "collapsible-menu",
         "include-codeblock",
         "ace",
-        "callouts"
+        "callouts",
+        "robots-txt@git+https://github.com/TugboatQA/gitbook-plugin-robots-txt.git#1.0.1"
```

Comment:
I think I'd rather have apache serve up a simple robots.txt from the docs vhost instead of maintaining a plugin, even if it is a simple plugin. We could also reuse that for the other vhosts.

Reply:
That's fine…although, done is beautiful?

Reply:
https://github.com/Lullabot/tugboat/pull/2344